### PR TITLE
[13.0] sale_delivery_state: backport to v13

### PR DESCRIPTION
**Pull request by simahawk**
Based on #1372

### CODE REVIEW

File: sale_automatic_workflow/__manifest__.py [hunk 1]
```diff
@@ -5,15 +5,15 @@
...
+    "version": "13.0.1.3.0",
     "category": "Sales Management",
     "license": "AGPL-3",
     "author": "Akretion, "
     "Camptocamp, "
     "Sodexis, "
     "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/sale-workflow",
-    "depends": ["sale_stock", "sales_team"],
+    "depends": ["sale_stock", "sales_team", "sale_delivery_state"],
```
Comment (pedrobaeza) [inline sale_automatic_workflow/__manifest__.py]: Please don't add such new dependency.

**Review by pedrobaeza — DISMISSED** [review]

File: sale_automatic_workflow/__manifest__.py (hunk 1 above)
Comment (simahawk) [reply sale_automatic_workflow/__manifest__.py]: we'll do this change in a glue module

Comment (simahawk) [reply sale_automatic_workflow/__manifest__.py]: solved

**Comment by pedrobaeza** [comment]
See my comment about the module name in 14.0

**Review by sebastienbeau — APPROVED** [review]
LGTM (code review)
this modules have been merge in V14

**Comment by simahawk** [comment]
rebased on v14 work. @PierrickBrun can you review? @pedrobaeza can you review or dismiss?

**Comment by PierrickBrun** [comment]
@simahawk Any idea why travis is red ? The error does not seem to be the same as the one in the `13.0` branch

**Comment by pedrobaeza** [comment]
Have you rebuilt? I pushed a fix for `sale_global_discount`.

**Comment by pedrobaeza** [comment]
Although I clicked incidentally on the close and comment button, it serves for relaunching Travis.

**Pull request merged**